src/libraries/System.Private.Xml/src/System/Xml/Core/XmlWriterAsync.cs

```diff
@@ -599,5 +599,20 @@ private async Task WriteLocalNamespacesAsync(XPathNavigator nsNav)
                 await WriteAttributeStringAsync("xmlns", prefix, XmlReservedNs.NsXmlNs, ns).ConfigureAwait(false);
             }
         }
+
+        public ValueTask DisposeAsync()
+        {
+            GC.SuppressFinalize(this);
+            return DisposeAsyncCore();
+        }
```

Comment:
```suggestion
    public async ValueTask DisposeAsync()
    {
        await DisposeAsyncCore();
        Dispose(false);
        GC.SuppressFinalize(this);
    }
```

Reply:
Thanks! Updated

Reply:
Stephen noted in a previous comment that disposeasync should use configureawait(false). 
Am I correct that this applies to the disposeasynccore call too?
``await DisposeAsyncCore().ConfigureAwait(false)``